src/lib/y2partitioner/actions/import_mount_points.rb

```diff
@@ -0,0 +1,91 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast/i18n"
+require "yast2/popup"
+require "y2partitioner/actions/controllers/fstabs"
+require "y2partitioner/dialogs/import_mount_points"
+
+module Y2Partitioner
+  module Actions
+    # Action for importing mount points from a fstab file
+    class ImportMountPoints
```

Comment:
NP: Not sure if we need a separate Actions object and a separate Controller for this. At first sight, it may look like a bit too much for not so much logic (is not one of those actions with multiple interdependent steps and so on).

Just thinking aloud, not implying is necessarily  wrong.

Reply:
Well, IMHO too much logic for only a button and a dialog. I guess the result is much clearer with the Action and the Controller. It is true that there aren't several steps, but there are several widgets accessing to common data. Moreover, the button would require info about the found fstabs to show an error popup or not, so it should read somehow the fstabs. I guess it is better to have all that stuff in a specific action instead of directly in the button.

Reply:
Ok. Fine with me.